tests/plugins/test_distribution_scope.py

```diff
@@ -66,13 +66,14 @@ def test_distribution_scope_allowed(self, tmpdir, base_from_scratch, parent_scop
         if base_from_scratch:
             allowed = True
         if allowed:
-            with caplog.at_level(logging.ERROR):
+            with caplog.at_level(logging.DEBUG):
```

Comment:
It's about this code below:
```python
if base_from_scratch:
    assert "no distribution scope set for" in caplog.text
```
Previously the assert was missing, when I added it the test failed because that message is logged at DEBUG level, not ERROR.

Reply:
ok